tests/sv/person_HassGetState.yaml

```diff
@@ -0,0 +1,65 @@
+language: en
+tests:
+  - sentences:
+      - "var är Daniel"
+    intent:
+      name: HassGetState
+      slots:
+        domain: person
+        name: Daniel
+    response: "Daniel är hemma"
+
+  - sentences:
+      - "var är Ola-Conny"
+    intent:
+      name: HassGetState
+      slots:
+        domain: person
+        name: Ola-Conny
+    response: "Ola-Conny är på jobbet"
+
+  - sentences:
+      - "är Judith"
```

Comment:
"är Judith hemma" ? 
Not sure that "är Judith" is a full command and in that case it might not be what we want to test.

Reply:
It was not. I added a zone (home).